install/preview/entrypoint.sh

```diff
@@ -143,8 +143,12 @@ run_telemetry(){
   sleep 100
   # indefinitely wait for Gitpod pods to be ready
   kubectl wait --timeout=-1s --for=condition=ready pod -l app=gitpod,component!=migrations
-  # manually tun the cronjob
-  kubectl create job gitpod-telemetry-init --from=cronjob/gitpod-telemetry
+  echo "Gitpod pods are ready"
+  # honour DO_NOT_TRACK if set
+  if [ -z "${DO_NOT_TRACK}" ]; then
+    # manually tun the cronjob
+    kubectl create job gitpod-telemetry-init --from=cronjob/gitpod-telemetry
```

Comment:
had to move this inside as `run_telemetry` also is useful for readyiness and showing that Gitpod is ready.